### ISSUE TITLE
Performance monitoring

### ISSUE THREAD

**Issue by dvdstelt** [Particular/NServiceBus.Transport.Bridge#9]
# Problem description

We should add performance monitoring (metrics) and add that to the platform as well.

Would need somewhat custom made solution, as we can't just add current metrics to the Bridge. We're creating proxies for actual existing endpoints and the platform would not be able to differentiate them.  
Also, things like `Critical time` don't really have meaning. It's even possible the Bridge screws up the actual times?

**Comment by Floopy-Doo** [comment]
We were looking into the option of adding metrics to our bridge as well. 
We would support this being added.

**Comment by SzymonPobiega** [comment]
It would be even better if the monitoring included uptime/liveliness like the heartbeats.

**Comment by SeanFeldman** [comment]
> It would be even better if the monitoring included uptime/liveliness like the heartbeats.

Correct. Being a part of the platform, monitoring with the platform would be the right approach, removing the need to build a custom solution each time messaging bridge is used.

**Comment by danielp37** [comment]
Has there been any progress on this issue? We are implementing the MessagingBridge and I'm looking for ways to be able monitor its performance and progress.
